examples/wifi-echo/server/esp32/main/gen/gen_config.h

```diff
@@ -118,18 +272,60 @@
 // Use this macro to check if Radio Coexistence Stub plugin is included
 #define EMBER_AF_PLUGIN_COEXISTENCE_STUB
 
+// Use this macro to check if Color Control Cluster Server plugin is included
+#define EMBER_AF_PLUGIN_COLOR_CONTROL_SERVER
+// User options for plugin Color Control Cluster Server
+#define EMBER_AF_PLUGIN_COLOR_CONTROL_SERVER_XY
+#define EMBER_AF_PLUGIN_COLOR_CONTROL_SERVER_TEMP
+#define EMBER_AF_PLUGIN_COLOR_CONTROL_SERVER_HSV
```

Comment:
This might cause @bhaskar-apple issues with bringing in the color control cluster.  If it does, we can comment these out at that point.